hash/oric1_cass.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="oric1" description="Oric 1 cassettes">
```

Comment:
The short name should match the file name – I think it fails a validity check if it doesn’t.

Reply:
Done